### PR TITLE
Prepare for the next release, update dependencies

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

**Comment by carlosthe19916** [comment]
I thought we were releasing on Thursdays. Do you mind giving me some time to verify the UI is not broken after the last set of PRs merged? This release includes a UI served through the server so it would be nice to have it working

**Comment by ctron** [comment]
> I thought we were releasing on Thursdays. Do you mind giving me some time to verify the UI is not broken after the last set of PRs merged? This release includes a UI served through the server so it would be nice to have it working

Indeed we do. I was just preparing for that. We can always add another PR to update the UI again.

File: Cargo.lock [hunk 1]
```diff
@@ -7192,7 +7192,7 @@ dependencies = [
 [[package]]
 name = "trustify-ui"
 version = "0.1.0"
-source = "git+https://github.com/trustification/trustify-ui.git?tag=static-main#b2d81aec3218a900bbb73bcdb69d483e96cbc48f"
+source = "git+https://github.com/trustification/trustify-ui.git?tag=static-main#b467f98f72dcd12ec2a82c34da48388a5e050618"
```
Comment (carlosthe19916) [inline Cargo.lock:7195]:
```suggestion
source = "git+https://github.com/trustification/trustify-ui.git?tag=static-main#857f4bfbc59c668fd1379360d7378c18df3f7f9a"
```

Comment (carlosthe19916) [reply Cargo.lock:7195]: I have added a commit to the ui in order to reflect the latest changes about /api/v1/package -> /api/v1/purl so the UI works as it used to work until this Monday demo.

**Review by carlosthe19916 — CHANGES_REQUESTED** [review]

**Comment by ctron** [comment]
Updated it.

**Review by carlosthe19916 — APPROVED** [review]

**Pull request merged**